README.md

```diff
@@ -1,24 +1,235 @@
+  -Code Editor
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Enter this url: git@github.com:uchexm/hello-rails-react.git in your web browser.
+Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+Select the download Zip option from drop-down menu.
+Once the download is complete you will be able to access my project locally.
+<!--
+Example commands:
+
+```bash
+$ git clone git@github.com:uchexm/hello-rails-react.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+  ```bash
+- bundle install
+- npm install
+```
+--->
+
+### Usage
+
+  Create Database (Mandatory)
+
+```bash
+rails db:create
+rails db:schema:load
+```
+
+If you have made a migration the run this command
+
+```bash
+rails db:migrate
+rails s
+``` 
+<!--
+Example command:
+
+```sh
+  rails server
+  yarn build --watch
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+
+ ```bash
+rspec
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+Heroku
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Uchechi Nwaka**
+
+- GitHub: [@uchexm](https://github.com/uchexm)
+- Twitter: [@nwakauc](https://twitter.com/nwakauc)
+- LinkedIn: [Uchechi Nwaka](https://linkedin.com/in/uchechi-nwaka-79a21a75/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- Add more endpoints
+- Display Date and Time
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/uchexm/hello-rails-react/issues.)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project...
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I ascribe all Glory to the Almighty God.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Rails and React is not soo difficult to learn]**
+
+  - [You need to be conversant with the necessary GEMS]
+
+- **[BUilding Rails API to an existing app]**
+
+  - [Create a new API controller with its route config in routes.rb]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job working on this milestone. When I click on the MIT license link, I am re-directed to an error 404 page. Kidly fix this as it not professional to allow broken links in our README

Reply:
Fixed.